tests/test_feedexport.py

```diff
@@ -843,3 +846,17 @@ def test_init_exporters_storages_with_crawler(self):
         yield self.exported_data({}, settings)
         self.assertTrue(FromCrawlerCsvItemExporter.init_with_crawler)
         self.assertTrue(FromCrawlerFileFeedStorage.init_with_crawler)
+
+    @defer.inlineCallbacks
+    def test_pathlib_uri(self):
+        tmpdir = tempfile.mkdtemp()
+        feed_uri = Path(tmpdir) / 'res'
+        settings = {
+            'FEED_FORMAT': 'csv',
+            'FEED_STORE_EMPTY': True,
+            'FEED_URI': feed_uri,
```

Comment:
Shouldn’t this be a file path instead of a folder path?

Reply:
According to previous tests, this is the manner in which they were done. I am unable to understand what you mean.
Thefeed_uri, from my understanding, is the file path, which the spider writes to.

Reply:
Nevermind, my bad.